assembly-main/src/assembly/bin/che.sh

```diff
@@ -482,42 +493,57 @@ function call_catalina {
   fi
 }
 
-function stop_che_server {
+stop_che_server () {
 
   if ! ${USE_DOCKER}; then
     echo -e "Stopping Che server running on localhost:${CHE_PORT}"
     call_catalina >/dev/null 2>&1
   else
-    echo -e "Stopping Che server running in docker container named ${GREEN}che${NC}."
+    echo -e "Stopping Che server running in docker container named ${GREEN}${CONTAINER}${NC}..."
 
-    DOCKER_EXEC="sudo service docker stop && /home/user/che/bin/che.sh stop"
-    "${DOCKER}" exec che ${DOCKER_EXEC} &>/dev/null 2>&1 || DOCKER_EXIT=$? || true
+    echo -e "Stopping docker container named ${GREEN}${CONTAINER}${NC}...."
```

Comment:
One is for stopping Che.  The other is stopping the container.  Necessary for debugging.